### ISSUE TITLE
KMCX

### ISSUE THREAD

**Issue by kheerdink** [julysfire/MSFS2020_Indiana_Megapack#2]
Terrain data has been improved but airports specifically KMCX has not been changed at all. Could it be that I need to modify its folder in any way?

**Comment by julysfire** [comment]
Have you added the KMCX folder to your community folder? The airport folded themselves contain all the scenery where the vegterrain is only the vegetation and terrain.
I believe I have updated the packages to the latest MSFS version but I will double check and do some further testing tonight when I can.

**Comment by kheerdink** [comment]
Perhaps I made a mistake? I added the entire zipped folder to the community
folder.

Thank you for the reply! Care to share your airport list that you are
working on? Is FKR Frankfort on it by chance?

Thanks again!

**Comment by julysfire** [comment]
Is it the zip folder that is in the community folder?  It should be unzipped contents.  I just uploaded an updated [zip folder ](https://github.com/julysfire/MSFS2020_Indiana_Megapack/blob/main/Indiana%20Megapack.zip) that has all the most recent updates for the airports.

When you unzip it to your community folder, it should look like this:
![image](https://user-images.githubusercontent.com/50681646/115132206-93ad9d80-9fc3-11eb-93f7-44ec1b74e45f.png)

As for the list of airports, I do have a laundry list of airports I would like to work on such as KGGP, KTYQ, KRZL, KBFR, KMGC and 05C.  That isn't a full list but I have added KFKR to the list as well.
